flit/inifile.py

```diff
@@ -117,7 +117,12 @@ def read_pkg_ini(path):
     if 'description-file' in md_sect:
         description_file = path.parent / md_sect.pop('description-file')
         with description_file.open() as f:
-            md_dict['description'] = f.read()
+            raw_desc =  f.read()
+        from readme.rst import render
+        _, ok = render(raw_desc)
+        if not ok:
+            raise ValueError("The file description seem to not be valid rst for PyPI")
```

Comment:
Let's make this a `log.warn()`, not an exception:

1. Descriptions don't *have* to be rst - plain text is valid, that's why it silently falls back to unrendered text if there's an error producing rst.
2. This shouldn't prevent it from installing, where the description doesn't matter.